internal/aws/proxy/server_test.go

```diff
@@ -53,9 +52,14 @@ func TestHappyCase(t *testing.T) {
 	cfg.ProxyAddress = "https://example.com"
 	srv, err := NewServer(cfg, logger)
 	assert.NoError(t, err, "NewServer should succeed")
-	go srv.ListenAndServe()
+	go func() {
+		_ = srv.ListenAndServe()
```

Comment:
```suggestion
		require.NoError(t, srv.ListenAndServe())
```

Reply:
I tried like this, but the test case doesn't work